src/drive/web/modules/views/Upload/UploaderComponent.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { render, RenderResult, waitFor } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { DumbUpload } from 'drive/mobile/modules/upload'
+import { generateForQueue } from './UploadUtils'
+
+jest.mock('cozy-keys-lib', () => ({
+  withVaultClient: jest.fn().mockReturnValue({})
+}))
+
+const tSpy = jest.fn()
+const uploadFilesFromNativeSpy = jest.fn()
+
+describe('DumbUpload component', () => {
+  const defaultItems = [{ fileName: 'File1.pdf' }]
+
+  const setupComponent = (): RenderResult<
+    typeof import('/home/anc/cozy/cozy-drive/node_modules/@testing-library/dom/types/queries'),
```

Comment:
to remove ?

Reply:
Agree, this shouldn't be merged until we fix this line